docs/naps/index.md

```diff
@@ -0,0 +1,20 @@
+(nap_list)=
+# napari advancement proposals (NAPs)
```

Comment:
```suggestion
# napari Advancement Proposals (NAPs)
```

Doc style nit: make casing of section titles consistent, though napari is a special lowercase-only word here.